Source/App/EbAppConfig.c

```diff
@@ -152,7 +152,10 @@ static void SetNaluFile(const char *value, EbConfig_t *cfg)
 {
     if (cfg->naluFile) { fclose(cfg->naluFile); }
     FOPEN(cfg->naluFile, value, "rb");
-    cfg->useNaluFile = EB_TRUE;
+    if (cfg->naluFile)
+        cfg->useNaluFile = EB_TRUE;
+    else 
+        printf("Nalu file: %s does not exist, won't use.", value);
```

Comment:
Thanks for your contribution.
Please add "\n" at the end.
I think it is better to add "Error: " at the beginning.